lib/services.js

```diff
@@ -72,10 +74,19 @@ module.exports = function generateServices(app, options) {
 
 function describeModels(app, options) {
   var result = {};
+  var modelsToIgnore = options.modelsToIgnore;
+
   app.handler('rest').adapter.getClasses().forEach(function(c) {
     var name = c.name;
     c.description = c.sharedClass.ctor.settings.description;
 
+    if(modelsToIgnore.indexOf(name) >= 0){
+      // Skip classes that are provided in options.modelsToIgnore array
+      // We don't want to publish these models in angular app
+      console.error('Skipping %j model as it is not to be published', name);
```

Comment:
IMO, `console.error` should be used to log error messages, which is not the case here. Why is it needed to print a message when we are just following the instructions provided by the user?